io/prometheus/client/metrics.proto

```diff
@@ -78,26 +80,27 @@ message Histogram {
   // then each power of two is divided into 2^n logarithmic buckets.
   // Or in other words, each bucket boundary is the previous boundary times 2^(2^-n).
   // In the future, more bucket schemas may be added using numbers < -4 or > 8.
-  optional sint32 schema           = 5;
-  optional double zero_threshold   = 6; // Breadth of the zero bucket.
-  optional uint64 zero_count       = 7; // Count in zero bucket.
-  optional double zero_count_float = 8; // Overrides sb_zero_count if > 0.
+  optional sint32                    schema             = 5;
+  optional double                    zero_threshold     = 6; // Breadth of the zero bucket.
+  optional uint64                    zero_count         = 7; // Count in zero bucket.
+  optional double                    zero_count_float   = 8; // Overrides sb_zero_count if > 0.
```

Comment:
If fields are separated by multi-line comments or even blank lines, I don't think it help readability to align them with a lot of whitespace.

Reply:
Yeah, I was kinda confused with the layout and thought it was made by some binary that I don't have installed. I'll revert these spacings 👍